### ISSUE TITLE
Always returns "Unauthenticated" for NextAuth session even though the credentails are correct

### ISSUE THREAD

**Issue by gimnathperera** [dejwid/food-ordering#2]
Always returns "Unauthenticated" for NextAuth session even though the credentials are correct. 
<img width="237" alt="image" src="https://github.com/dejwid/food-ordering/assets/55834384/3b4b0c37-f5c8-4333-afc3-c0babef5e472">

**Comment by rasif-sahl** [comment]
yes, I am also having the same issue @gimnathperera did you find the issue I tried multiple things but couldn't find the exact issue. 

@dejwid can you help us with this

**Comment by gimnathperera** [comment]
Hi @rasif-sahl yes I found the issue. the issue is with the `@auth/mongodb-adapter`. And you actually don't need it here. If you remove the `mongodb-adapter` usage and re-try it'll work.

**Comment by rasif-sahl** [comment]
Ohh yes, thanks @gimnathperera that was the issue... But any particular reason for this

**Comment by seharnazeer** [comment]
Hey this happens because we have to enable JSON web token as session for the credentials provider are not presisted in the database ,just add these lines after providers . Things will start working . Thanks
```
  session: {
    strategy: "jwt",
    maxAge: 30 * 24 * 60 * 60 // 30 days
  }
```

**Comment by asfiiiii** [comment]
Thank you, @seharnazeer. I have a question regarding our current practice of storing sessions in JWT instead of a database. I am curious about potential implications on our application workflow and whether we may encounter any issues as a result. Thus far, my findings suggest that sessions can be stored either in a database or using JWT.
